content/source/docs/enterprise/private/vmware-setup-guide.html.md

```diff
@@ -30,13 +30,12 @@ Depending on the chosen [operational
 mode](https://www.terraform.io/docs/enterprise/private/install-installer.html#operational-mode-decision),
 the infrastructure requirements for PTFE range from a single virtual machine
 for demo or proof of concept installations, to multiple virtual machines
-hosting postgres, access to S3 or Azure storage, and external Vault servers for
+hosting postgres, access to S3, Azure or Google Cloud storage, and external Vault servers for
```

Comment:
This parses kind of weird for me.  Maybe a comma after "Azure", but also, they don't need a VM for "access to S3"; they'd either need the VM to _have_ access, or to provide their own storage service, like ceph.

Reply:
Yeah... that is odd, actually... maybe "to multiple virtual machines hosting the Terraform Enterprise application, postgres and external Vault servers for..." ?

Reply:
yeah.  Also, I think @armchairlinguist  mentioned this earlier, but we should probably use "PostgreSQL", as well.